packages/language-chooser/src/language-chooser/stories/index.story.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import type { Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import LanguageChooser from '../';
+import type { Language } from '../types';
+
+const meta: Meta< typeof LanguageChooser > = {
+	title: 'Components/Language Chooser',
```

Comment:
The first token in the path is usually the package name

```suggestion
	title: 'Language Chooser/Language Chooser',
```

Reply:
Yeah it's just a remnant from the back and forth in this PR. I had it like this before.